src/util/FormField.js

```diff
@@ -224,7 +224,7 @@ const FormTimestampField = ({
                 variant="outlined"
                 error={!!errorMsg}
                 required={required}
-                value={date ? format(date).format("HH:mm:ss") : ""}
+                value={date ? formatDateObject(date, "YYYY-MM-DD") : ""}
```

Comment:
This changes the format this `time` text field from `hours:minutes:seconds` to `year-month-day`.

Reply:
Heh oops. I got confused with variable name date i guess but anyaways, `format(date).format("HH:mm:ss")` is not a function in `datefns` lib.

Reply:
🤔 I guess it's time to find a combined date/time picker then, instead of these separate, native `date` and `time` pickers. These native form fields are meant to update only a single "timestamp" field anyway.

Reply:
We do have `date and time` pickers in MUI - https://material-ui.com/demos/pickers/

Reply:
This form field is currently a combination of the MUI native [Date picker](https://material-ui.com/demos/pickers/#date-pickers) and [Time picker](https://material-ui.com/demos/pickers/#time-pickers) examples.

Their native [Date & Time picker](https://material-ui.com/demos/pickers/#date-amp-time-pickers) example uses `type="datetime-local"`, which isn't very functional in Firefox, since it renders as a simple text field without any UI controls or validation.

Reply:
May be https://www.npmjs.com/package/react-datetime-picker ?

Reply:
I'm probably missing something coming into the convo a bit late, but `format(new Date(), 'HH:mm:ss')` seems to work for me, so I would think `formatDateObject(date, 'HH:mm:ss')` would work as well?

Reply:
Correct. I don' think I got your question 🤔

Reply:
Oh ok, I'm probably not understanding the issue.  I thought the issue was that date-fns won't take a date and convert it to `HH:mm:ss`, and because of that it became discussion about alternatives to date-fns?

Reply:
Ah. Originally I had `format(date).format("HH:mm:ss")`which is not valid in `date-fns`. So I refactored with another function in `DateFormatter.js` that still uses `date-fns`. So `formatDateObject` is still using `date-fns` under the cover :). Sorry for the confusion.

Reply:
Yeah, that was also my (mis?)understanding...

> but anyaways, `format(date).format("HH:mm:ss")` is not a function in `datefns` lib.

Reply:
> Ah. Originally I had `format(date).format("HH:mm:ss")`which is not valid in `date-fns`. So I refactored with another function in `DateFormatter.js` that still uses `date-fns`. So `formatDateObject` is still using `date-fns` under the cover :). Sorry for the confusion.

🆗 In that case, I don't think we have to worry about refactoring the date/time picker to something else for this PR.